specs/interop/shared-lockbox.md

```diff
@@ -0,0 +1,340 @@
+# Shared Lockbox
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Design](#design)
+  - [Contracts inroductions](#contracts-inroductions)
+    - [`SharedLockbox`](#sharedlockbox)
+    - [`LiquidityMigrator`](#liquiditymigrator)
+  - [Contracts upgrades](#contracts-upgrades)
+    - [`SuperchainConfig`](#superchainconfig)
+    - [`OptimismPortal`](#optimismportal)
+- [Reference implementation](#reference-implementation)
+  - [`SharedLockbox`](#sharedlockbox-1)
+  - [`LiquidityMigrator`](#liquiditymigrator-1)
+  - [`SuperchainConfig`](#superchainconfig-1)
+  - [`OptimismPortal`](#optimismportal-1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+With interoperable ETH, withdrawals may fail if the referenced `OptimismPortal` lacks sufficient ETH
+— making it mandatory to withdraw from the L2 where the ETH was originally deposited.
```

Comment:
```suggestion
— making it mandatory to withdraw from the L2 where the ETH was originally deposited.
```

does this make sense? i don't fully get it, why would it be mandatory?

Reply:
I think it refers to an alternative approach to the lockbox in which the ETH per L2 is tracked on the portal, is that right @agusduha?

But maybe is a good idea to remove this sentence, I think the paragraph is clear enough without it